src/backend/rest.ts

```diff
@@ -72,10 +72,19 @@ export default class RestApiBackend implements Backend {
       url: '/version',
       method: 'GET',
     });
-    // TODO: check if this version matches the edition!
+    const checkMkVersion: string = result.data.versions.checkmk;
+    if (checkMkVersion.startsWith("2.0") || checkMkVersion.startsWith("1.6")) {
+      throw new Error(`CheckMK version ${checkMkVersion} is not supported for this plugin, but you can set the backend to the '< 2.2' version and use at your own risk.`)
```

Comment:
i would not use `${checkMkVersion}` but the if condition: `<2.1`. if we have an error in the version comparison we would get a bug report, because the used version and the error message does not match.

also `Checkmk` instead of `CheckMK`

(both comments also apply to the next error message)